test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -287,6 +289,23 @@ void test_ucp_peer_failure::do_test(size_t msg_size, int pre_msg_count,
 
     EXPECT_EQ(UCS_OK, m_err_status);
 
+#if HAVE_IB
+    /* Since UCT/UD EP has a SW implementation of reliablity on which peer
+     * failure mechanism is based, we should set small UCT/UD EP timeout
+     * for UCT/UD EPs for sender's UCP EP to reduce testing time */
+    ucp_ep_config_t *ep_config = ucp_ep_config(failing_sender());
+    for (ucp_lane_index_t lane = 0; lane < ep_config->key.num_lanes; ++lane) {
```

Comment:
for simplicity: we can just go over all rsc_index values. no need to limit ourselves to the lanes in ucp_ep

Reply:
> for simplicity: we can just go over all rsc_index value

good catch! done